README.md

```diff
@@ -48,6 +51,34 @@ pacman -S base-devel git ncurses lib32-zlib gawk time unzip perl-xml-libxml \
  flex wget gettext quilt python2 openssl
 ```
 
+
+### Build-System-Structure  
+
+Where can I change something? What does these files do? 
```

Comment:
isn't is "What do these files do" or "What does a file do"?

Reply:
right. Wasn't quite sure on that one.